packages/acceptit-Tests.package/ACStepParsingTest.class/instance/testItParsesTrue.st

```diff
@@ -0,0 +1,11 @@
+as yet unclassified
+testItParsesTrue
+	
+	|ac method|
+	ACReadMeFactory executeReadMe.
+	ac := (Smalltalk classNamed: 'ACCalculatorUserStory') new.
+	method := ac methodForVerb: #then andStep: 'I expect A to be true'.
```

Comment:
A Question concerning testing in general. Here we test whether the parsing of this single example is right and use this result to validate our parser. But honestly anything except this example could yield bullshit and the test still would be green. What we need to do is to test the parsing rules in general. But I cant really figure out how this could be done

Reply:
Unit testing tests examples. It's good if you have multiple examples (although you don't want to over do it) but to validate the whole logic we'd have to recode the logic into the tests which then doesn't test anything because it validates the logic with itself.